Update README.md

Swennet opened the pull request:
I found some small issues in the current readme which will probably confuse beginners. I updated the readme to fix one small issue, and updated the example for the Upscale method.

Swennet committed:
Update README.md

Swennet committed:
Update README.md

Swennet committed:
Update README.md

Swennet committed:
Update README.md

Swennet committed:
Update README.md

ShuCh3n commented on README.md:
```diff
@@ -103,13 +103,24 @@ $result = $imagine_builder->send()
+// After the initial imagine interaction has completed, we can use the result to request an upscaled version.
+$upscale_image = 3; // Select the 3rd generated image for upscaling
+
+preg_match('/\[(.*?)\]/', $result['content'], $matches);
```
You don't need to do a preg_match. The builder object generates an interaction id on the construct. 

```php
$interaction_id = $result->interactionId() //You can retrieve this ID after the imagine interaction is performed, this is a identifier for the specific job request.
```

Swennet committed:
Update README.md

Remove preg_match in favor of $result->interactionId

Swennet committed:
Delete .DS_Store

Swennet committed:
Update README.md

Swennet commented:
@ShuCh3n Ah, I see what I did wrong previously when I tried to use `$result->interactionId()`. Using `->send()` always returns the array, and not the original Imagine class which has the interactionId.

I've updated the readme for Upscale to reflect that. (And also deleted a stray .DS_Store file)

adas-hitpromo commented:
I am getting error like this.

eDiasoft\Midjourney\Exceptions\MidjourneyException
Can't retrieve Guild ID.
http://ssm.com/getImageUrl
